certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7804/FK7804SickLeaveProvider.java

```diff
@@ -29,9 +31,14 @@ public Optional<SickLeaveCertificate> build(Certificate certificate) {
         .map(ElementValueBoolean::value)
         .orElse(false);
 
-    if (isNotSickLeaveCertificate) {
+    if (isNotSickLeaveCertificate && !ignoreModuleRules) {
       return Optional.empty();
     }
+    return buildSickLeaveCertificate(certificate, ignoreModuleRules);
+  }
+
+  public Optional<SickLeaveCertificate> buildSickLeaveCertificate(Certificate certificate,
```

Comment:
What do you think about making this private? I noticed some unit tests use it but they could just use the build(certificate, ignoreModuleRules) method instead.